INTRODUCTION.md

```diff
@@ -132,6 +132,52 @@ If you are using Visual Studio Code, we recommend [Red Hat's YAML extension](htt
 ```
 to your `settings.json` file, you will get help from the editor on YAML files following the namepatterns to the right (might have to restart the editor after updating the settings).
 
+#### Logging
+
+The default logging configuration for a Webviz application is to only show log messages with level `WARNING` or `ERROR`.
+This global logging level can be changed on the command line using the `--loglevel` argument like this:
+```bash
+webviz build ./examples/basic_example.yaml --loglevel INFO
+```
+
+In practice, setting the global logging level to `INFO` or `DEBUG` will likely flood the log with messages from various 
+sub-modules, possibly obscuring messages from the module(s) of interest. To allow finer grain control over logging, 
+Webviz accepts the `--logconfig` command line argument which allows detailed logging configuration through a YAML file:
+```bash
+webviz build ./examples/basic_example.yaml --logconfig my_log_config.yml
```

Comment:
Should we keep a consistent file extension for YAML in the command example? `.yaml` or `.yml`?

Reply:
Absolutely. Since the `--logfile` parameter in the example refers to a imaginary file I'll just change the extension to `.yaml`